airflow/ui/src/pages/Dag/Runs/Runs.tsx

```diff
@@ -92,6 +93,18 @@ const columns: Array<ColumnDef<DAGRunResponse>> = [
       `${dayjs.duration(dayjs(original.end_date).diff(original.start_date)).asSeconds().toFixed(2)}s`,
     header: "Duration",
   },
+  {
+    accessorKey: "clear_dag_run",
+    cell: ({ row }) => (
```

Comment:
Can we align the icon right?
(atm it is centered in a column that is wider than the icon)